x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecks.java

```diff
@@ -81,174 +74,14 @@ private static void fieldLevelMappingIssue(IndexMetaData indexMetaData, BiConsum
-    }
-
-    static DeprecationIssue indexStoreTypeCheck(IndexMetaData indexMetaData) {
-        if (indexMetaData.getCreationVersion().before(Version.V_6_0_0_alpha1) &&
-            indexMetaData.getSettings().get("index.store.type") != null) {
-            return new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
-                "The default index.store.type has been removed. If you were using it, " +
-                    "we advise that you simply remove it from your index settings and Elasticsearch" +
-                    "will use the best store implementation for your operating system.",
-                "https://www.elastic.co/guide/en/elasticsearch/reference/6.0/" +
-                    "breaking_60_settings_changes.html#_store_settings", null);
-
-        }
-        return null;
-    }
-
-    static DeprecationIssue storeThrottleSettingsCheck(IndexMetaData indexMetaData) {
-        if (indexMetaData.getCreationVersion().before(Version.V_6_0_0_alpha1)) {
-            Settings settings = indexMetaData.getSettings();
-            Settings throttleSettings = settings.getAsSettings("index.store.throttle");
-            ArrayList<String> foundSettings = new ArrayList<>();
-            if (throttleSettings.get("max_bytes_per_sec") != null) {
-                foundSettings.add("index.store.throttle.max_bytes_per_sec");
-            }
-            if (throttleSettings.get("type") != null) {
-                foundSettings.add("index.store.throttle.type");
-            }
-
-            if (foundSettings.isEmpty() == false) {
-                return new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
-                    "index.store.throttle settings are no longer recognized. these settings should be removed",
-                    "https://www.elastic.co/guide/en/elasticsearch/reference/6.0/" +
-                        "breaking_60_settings_changes.html#_store_throttling_settings", "present settings: " + foundSettings);
-            }
-        }
-        return null;
-    }
-
-    static DeprecationIssue indexSharedFileSystemCheck(IndexMetaData indexMetaData) {
-        if (indexMetaData.getCreationVersion().before(Version.V_6_0_0_alpha1) &&
-            indexMetaData.getSettings().get("index.shared_filesystem") != null) {
+    static DeprecationIssue oldIndicesCheck(IndexMetaData indexMetaData) {
+        Version createdWith = indexMetaData.getCreationVersion();
+        if (createdWith.before(Version.V_6_0_0)) {
             return new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
-                "[index.shared_filesystem] setting should be removed",
-                "https://www.elastic.co/guide/en/elasticsearch/reference/6.0/" +
-                    "breaking_60_indices_changes.html#_shadow_replicas_have_been_removed", null);
+                "Index created before 6.0",
+                "https://www.elastic.co/guide/en/elasticsearch/reference/master/" +
+                    "breaking-changes-7.0.html#breaking_70_packaging_changes",
```

Comment:
```suggestion
                    "breaking-changes-7.0.html",
```

I think this link should just link to `https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking-changes-7.0.html` - the warning about indices created in old versions is at the top of the page and the text at this anchor doesn't really relate to this issue.